lib/fixme.ex

```diff
@@ -2,20 +2,26 @@ defmodule FIXME do
   defmacro fixme(
     {:-, _, [{:-, _, [year, month]}, day]},
     message,
-    opts \\ :default_opts
+    opts \\ []
   ) do
     {opts, _} = Code.eval_quoted(opts)
 
     # Injectable "today" for tests.
-    current_date = case opts do
-      :default_opts -> :calendar.local_time |> elem(0)
-      [today: date] -> date
-    end
+    current_date = opts[:today] || (:calendar.local_time |> elem(0))
+
+    # Opt into compile time warnings
+    warn = opts[:warn]
```

Comment:
I agree that off by default makes sense. What I was also trying to say is: what if you could do

``` elixir
config :fixme, warn: true
```

instead of having to specify `warn: true` each time? It seems reasonable to me that someone might want warnings or not, without having to specify it for each `fixme` call. I've used the Ruby version of this lib a lot, and I know I wouldn't want to type more than absolutely necessary.